src/languages/scala.js

```diff
@@ -132,6 +135,22 @@ export default function(hljs) {
     }
   };
 
+  const IMPORT_EXPORT = {
+    beginKeywords: 'import export',
+    end: /[;\n]/,
+    contains: [
+      { beginKeywords: 'as' },
+      {
+        begin: '{',
+        end: '}',
+        contains: [
+          TYPE,
+          { beginKeywords: 'as' },
+        ]
+      }
+    ],
+  };
```

Comment:
From looking at your example woud'n't this work just as well:

```suggestion
  const IMPORT_EXPORT = {
    beginKeywords: 'import export',
    end: /[;\n]/,
    contains: [
      { beginKeywords: 'as' },
      TYPE
    ],
  };
```

We try to be a simple pattern matcher (rather than a full parser) whenever we can - as it makes the grammars simpler, smaller, and easier to maintain.

Reply:
Ping.